content/doc/developer/publishing/plugin-site.adoc

```diff
@@ -1,8 +1,109 @@
 ---
 title: Plugin Site
 layout: developersection
-wip: true
-references:
-- url: https://wiki.jenkins.io/display/JENKINS/Plugin+Site+Information+for+Plugin+Developers
-  title: Plugin site information on the wiki
 ---
+
+This page explains how https://plugins.jenkins.io/[the plugins site]
+works. The intended audience is plugin developers.
+
+This is not a manual on how to _use_ the site to e.g. find plugins, but
+rather explains how it gets its data and how plugin developers can
+change the data on it.
+
+== Data shown on the plugins site
+
+=== Which plugins are listed? How can I make my plugin appear?
+
+All plugins published to the current (weekly) update center are listed.
+
+=== What is the plugin installation trend based on?
+
+This uses the monthly data from http://stats.jenkins.io/ as gathered
+from Jenkins instances that have not opted out of reporting data.
+
+Learn more: link:../usage-statistics[usage statistics]
+
+=== What does "most installed" mean?
+
+This ranks plugins by the number of different Jenkins instances that
+reported the plugin installed during the previous month.
+
+=== What does 'Recently updated' mean?
+
+This is based on metadata created by the update site generator, which
+uses the creation date of a plugin release.
+
+Note that very recent plugin releases may not appear here due to caching
+and delays in data aggregation.
+
+=== What does 'Trending' mean?
+
+This is an attempt to capture the plugins whose popularity is growing
+rapidly. As we only have the monthly installation stats for this, it's
+necessarily a worse approximation than what you'd likely see e.g. in app
+stores.
+
+Currently (2016-11) this just divides the previous month's install count
+by the month before that. Plugins with the greatest numbers are shown in
+descending order.
+
+=== What are categories based on?
+
+Categories are based on the labels on plugin wiki pages. Those with the
+`+plugin-+` prefix are passed through update site generation and will
+appear in update-center.json (which gets downloaded by Jenkins, where
+these labels also serve to group plugins in the Plugin Manager).
```

Comment:
This hasn't been true in years, so this looks like a nice opportunity to correct this.

Categories are currently based on https://github.com/jenkins-infra/update-center2/#categorizing-plugins